extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/TransferProcessApiController.java

```diff
@@ -146,6 +153,22 @@ public void deprovisionTransferProcess(@PathParam("id") String id) {
         }
     }
 
+    private DataRequest dataRequest(String assetId, TransferRequestDto object) {
```

Comment:
In my opinion assetId should be into `TransferRequestDto` and this method should be moved in a transformer that transforms `TransferRequestDto` in `DataRequest`

Reply:
@Izzzu I think it's like this because at first this api was on the asset one, but in my opinion asset id is just a parameter of the data request, not to be represented as the resource id in the "REST way"